corehq/apps/sms/management/commands/run_sms_queue.py

```diff
@@ -53,11 +66,27 @@ def enqueue(self, queued_sms):
     def handle(self, **options):
         while True:
             try:
-                self.create_tasks()
+                if settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS and not can_send_sms_now():
```

Comment:
rather than doing this I think it would be better to create a setting:

```
# start and end time tuple during which SMS can be sent. Set to `None` to allow sending at any time.
# Times should be specified in UTC
SMS_RESTRICTED_SENDING_TIME = ('03:30:00', '15:30:00')
```

Reply:
awesome! I will use that.